outbound/index.js

```diff
@@ -320,12 +320,12 @@ exports.build_todo = function (todo, ws, write_more) {
         }
     }
 
-    var todo_json = JSON.stringify(todo, exclude_from_json);
-    var buf = new Buffer(4 + todo_json.length);
+    const todo_json = JSON.stringify(todo, exclude_from_json);
+    const buf = new Buffer(4 + todo_json.length);
     buf.writeUInt32BE(todo_json.length, 0);
     buf.write(todo_json, 4);
 
-    var continue_writing = ws.write(buf);
+    const continue_writing = ws.write(buf);
     if (continue_writing) return process.nextTick(write_more);
 
     ws.once('drain', write);
```

Comment:
It doesn't appear write is defined. @baudehlo , should it have been renamed from write_more?